features/checkout/paying_for_order/preventing_not_available_payment_method_selection.feature

```diff
@@ -0,0 +1,22 @@
+@checkout
+Feature: Preventing not available payment method selection
+    In order to pay for my order properly
+    As a Customer
+    I want to be prevented from selecting not available payment methods
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And the store has "Offline" payment method
+        And the store has "Raven Post" shipping method with "€10.00" fee
```

Comment:
As it's not case of scenario, we should use ``And the store ships everywhere for free``.